src/main/java/plugily/projects/villagedefense/events/ChatEvents.java

```diff
@@ -72,20 +72,21 @@ public void onChatIngame(AsyncPlayerChatEvent event) {
           eventMessage = eventMessage.replaceAll(Pattern.quote(regexChar), "");
         }
       }
-      String message = formatChatPlaceholders(LanguageManager.getLanguageMessage("In-Game.Game-Chat-Format"), plugin.getUserManager().getUser(event.getPlayer()), eventMessage);
-      message = message.replace("%kit%", plugin.getUserManager().getUser(event.getPlayer()).getKit().getName());
-      if (plugin.getServer().getPluginManager().isPluginEnabled("PlaceholderAPI") && PlaceholderAPI.containsPlaceholders(message)) {
-        message = PlaceholderAPI.setPlaceholders(event.getPlayer(), message);
+      event.setMessage(ChatColor.stripColor(eventMessage));
+
+      String format = formatChatPlaceholders(LanguageManager.getLanguageMessage("In-Game.Game-Chat-Format"), plugin.getUserManager().getUser(event.getPlayer()));
+      if (plugin.getServer().getPluginManager().isPluginEnabled("PlaceholderAPI") && PlaceholderAPI.containsPlaceholders(format)) {
+        format = PlaceholderAPI.setPlaceholders(event.getPlayer(), format);
       }
```

Comment:
You can remove these lines here, its already in `formatChatPlaceholders` method.